lib/webpacker/compiler.rb

```diff
@@ -35,7 +35,8 @@ def compile
 
   # Returns true if all the compiled packs are up to date with the underlying asset files.
   def fresh?
-    watched_files_digest == last_compilation_digest
+    cached = last_compilation_digest
+    !cached.nil? && watched_files_digest == cached
```

Comment:
How about using `&.`?

```suggestion
    last_compilation_digest&.== watched_files_digest
```

Reply:
Thanks for your review. ❤️ 
I fixed it and force-pushed.

Reply:
Should I add `!!` to make the return value bool?
I'm a little worried because Ruby 3.0 has a function related to types.

```ruby
!!(last_compilation_digest&.== watched_files_digest)
```

Reply:
There is a `boolish` type so I don't need to cast it.
ref: https://github.com/ruby/rbs/blob/b98cd0afa9c575c8a97040b901909c93ec7d677d/docs/syntax.md#bool-or-boolish